.github/workflows/sentry.yaml

```diff
@@ -0,0 +1,51 @@
+name: Sentry Release
+
+on:
+  push:
+    branches:
+      - main
+      - develop
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
```

Comment:
Similar to what was done [here](https://github.com/NIAEFEUP/tts-fe/commit/9281efec35f4e8e0c2cf45d18994d1594680b088) in the `niployments` action, I think we need to state what is the environment we are using (main or develop) so that the action know which environment to fetch the secrets from.

This is different from the sentry environments

Reply:
Oh, I see. I thought that was automatically. So, I guess I will create a new environment for sentry variables and defined that here